Examples/Io/Csv/include/ActsExamples/Io/Csv/CsvMultiTrajectoryWriter.hpp

```diff
@@ -0,0 +1,77 @@
+// This file is part of the Acts project.
+//
+// Copyright (C) 2021 CERN for the benefit of the Acts project
+//
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+#pragma once
+
+#include "Acts/EventData/MultiTrajectoryHelpers.hpp"
+#include "ActsExamples/EventData/Trajectories.hpp"
+#include "ActsExamples/Framework/WriterT.hpp"
+#include "ActsFatras/EventData/Barcode.hpp"
+
+#include <fstream>
+
+using namespace Acts::UnitLiterals;
+
+namespace ActsExamples {
+
+/// @brief Struct for brief trajectory summary info
+/// @TODO: add nSharedHits
+///
+struct trackInfo : public Acts::MultiTrajectoryHelpers::TrajectoryState {
```

Comment:
Nitpicking: since this struct is used only by the `CsvMultiTrajectoryWriter`, I guess it's better to move it to the `private` of the `CsvMultiTrajectoryWriter`?

Reply:
Moved to the `private`